src/Pay.tsx

```diff
@@ -63,6 +63,7 @@ const Pay = () => {
         const tx = swapStatusTransaction();
 
         if (swap().asset === RBTC && tx && swap().claimTx === undefined) {
+            // @ts-ignore
```

Comment:
`swapStatusTransaction` has the wrong type

Reply:
fixed in follow up tsc pr's